src/services/kdbInsights/codeFlowLogin.ts

```diff
@@ -15,11 +15,12 @@ import axios from "axios";
 import * as crypto from "crypto";
 import * as fs from "fs-extra";
 import * as http from "http";
-import open from "open";
```

Comment:
quick fix, could you remove this package, since we are not using it anymore?

Reply:
Removed open package